src/swagger-specs/query-service.yaml

```diff
@@ -1825,4 +2174,578 @@ definitions:
         description: path to patch
       value:
         type: string
-        description: value of path
\ No newline at end of file
+        description: value of path
+  
+  subscribe_user_to_alert_body:
+    type: object
+    description: The request payload used to subscribe emailIds to query or schedule IDs for a given alert type.
+    required:
+      - assetId
+      - alertType
+      - subscriptions
+    properties:
+      assetId:
+        type: string
+        description: The query or schedule ID that you want the alert to track.
+        example: 'a679dd0e-bcb2-4e69-a610-22d17ba98cac'
+      alertType:
+        type: string
+        description: The state of query execution that you want to trigger an alert. The alert type is one of three values (`start`, `success`, `failure`).
+        example: 'failure'
+      subscriptions:
+        type: object
+        required:
+          - emailIds
+          - inContextNotifications
+          - emailNotifications
+        description: This object provides the users details that you want to receive the alert and how the users want to receive them. You must include their email ID and preferred notification channels for the alerts.
+        properties:
+          emailIds:
+            type: array
+            description: These are Adobe registered email addresses. The maximum number of `emailID`s that can be registered at once is five.
+            items:
+              $ref: '#/definitions/emailIdForSubscribing'
+          inContextNotifications:
+            type: boolean
+            description: A true value indicates that the users want to receive an alert through the Experience Platform UI. A false value would not provide an alert in the Adobe Experience Platform UI.
```

Comment:
For booleans we still backtick the values, and I usually phrase as "A value of `true`..." or "A value of `false`..." (Same thing on 2214)

Strange that you go with "Experience Platform UI" first, but the longer "Adobe Experience Platform UI" after? "Adobe Experience Platform" is mentioned in the intro paragraph for the swagger, so every other mention should just be "Experience Platform" or "Platform".

Don't use soft/indeterminate language like "would"/"could". APIs should be very predictable: "A value of `false` means that the users will not be sent alerts..." (Same thing on 2214)